source/includes/_feature-deriving.md

```diff
@@ -395,4 +395,158 @@ POST'ing to the private variables catalog a Shoji Entity containing a ZCL functi
 
 ### Other transformations
 
-Forthcoming.
+#### bin
+Receives a numeric column and returns a categorical column where each
+category represents a bin of the numeric values.
+Each category is annotated with a "boundaries" member that contains the 
+lower/upper bound of each bin.
+
+```json
+{
+    "function": "bin",
+    "args": [
+        {"variable": "http://.."}
+    ]
+}
+```
+
+#### case
+Returns a categorical column with its categories following the specified
+conditions from different columns on the dataset. [View Case Statements](#Case-statements)
+
+#### cast
+Returns a new column with its type and values casted. Not applicable
+on arrays or date columns.
```

Comment:
Can't you format dates as text somehow? Different function? (Just curious, not relevant for them.)